homeassistant/components/withings/manifest.json

```
@@ -3,7 +3,7 @@
   "name": "Withings",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/withings",
-  "requirements": ["withings-api==2.1.3"],
-  "dependencies": ["api", "http", "webhook"],
+  "requirements": ["withings-api==2.1.6"],
+  "dependencies": ["webhook", "http", "persistent_notification"],
```

Why do we need to depend on `"persistent_notification"`?
```suggestion
  "dependencies": ["http", "webhook"],
```